api/core/span_context.go

```diff
@@ -59,7 +59,13 @@ func (t TraceID) IsValid() bool {
 // MarshalJSON implements a custom marshal function to encode TraceID
 // as a hex string.
 func (t TraceID) MarshalJSON() ([]byte, error) {
-	return json.Marshal(hex.EncodeToString(t[:]))
+	return json.Marshal(t.String())
+}
+
+// Stringer Implementation of String() returns the hex string representation
+// form of a TraceID
```

Comment:
The function docs usually start with the name of the function, so:
```suggestion
// String returns the hex string representation form of a TraceID.
```

Reply:
Done.